docs/source/conf.py

```diff
@@ -24,7 +24,7 @@
 author = 'Philipp Wirth, Igor Susmelj'
 
 # The full version, including alpha/beta/rc tags
-release = '1.0.0'
+release = '1.2.0'
```

Comment:
You could try to get this on from `../lightly/__init__.py`

Reply:
Works :)